Update n1 waf docs and changelog to reference default log profiles an… (#2110)
nginx/documentation: content/nim/waf-integration/policies-and-logs/log-profiles/deploy-log-profile.md

1. Select the **Configuration** tab, then select **Edit Configuration**.

   {{< call-out class="note" >}}

>    Note: you can also reference a F5 WAF for NGINX [default logging profile]({{< ref "/waf/logging/log-overview.md#default-logging-profile-bundles" >}}) by using its name, no deployment required.

Reviewer: ```suggestion
  You can also reference an F5 WAF for NGINX [default logging profile]({{< ref "/waf/logging/log-overview.md#default-logging-profile-bundles" >}}) by name. You don't need to deploy it first.
```

Reviewer: You don't need "Note:" if you enclose the text in `{{< call-out class="note" >}}`
